Makefile

```diff
@@ -8,17 +7,29 @@
 .PHONY : docker-run docker-stop
 
 SHELL := /bin/bash
-VERSION ?= $(shell git describe 2> /dev/null)
-#VERSION := v0.1.0 # for debugging updater
+
+VERSION := $(shell git describe)
 
 RFC_3339 := "+%Y-%m-%dT%H:%M:%SZ"
 DATE := $(shell date -u $(RFC_3339))
```

Comment:
This needs to be removed. It's preventing `make` from being run on windows. We have a `DATE` var in the if statement below.